File: component_core/src/main/java/com/ashlikun/baseproject/libcore/mvp/view/BaseListPresenter.java
package com.ashlikun.baseproject.libcore.mvp.view;


import com.ashlikun.core.BasePresenter;
import com.ashlikun.core.iview.IBaseView;

import java.util.ArrayList;


/**
 * 作者　　: 李坤
 * 创建时间: 16:21 Administrator
 * 邮箱　　：496546144@qq.com
 * <p>
 * 功能介绍：
 */
public abstract class BaseListPresenter<D, V extends IBaseView> extends BasePresenter<V> {
    public ArrayList<D> listDatas = new ArrayList<>();
}

File: component_core/src/main/java/com/ashlikun/baseproject/libcore/mvp/view/BaseListPresenter.kt
package com.ashlikun.baseproject.libcore.mvp.view


import com.ashlikun.core.BasePresenter
import com.ashlikun.core.iview.IBaseView
import java.util.*


/**
 * 作者　　: 李坤
 * 创建时间: 16:21 Administrator
 * 邮箱　　：496546144@qq.com
 *
 *
 * 功能介绍：
 */
 abstract class BaseListPresenter<D, V : IBaseView> : BasePresenter<V>() {
    val listDatas = ArrayList<D>()
}
